src/Core/SpecificPrice/SpecificPriceFormatter.php

```diff
@@ -0,0 +1,182 @@
+<?php
```

Comment:
This class is an extraction of the legacy function **formatQuantityDiscounts**, it is still legacy code, I extracted it so I could pass it some values has dependency, especially the **isTaxIncluded** boolean, so it can be tested with and without tax.